sdk/keyvault/keyvault-certificates/src/lro/core-lro-update.ts

```diff
@@ -96,14 +100,11 @@ export abstract class KVPoller<TState extends PollOperationState<TResult>, TResu
     // This prevents the UnhandledPromiseRejectionWarning in node.js from being thrown.
     // The above warning would get thrown if `poller.poll` is called, it returns an error,
     // and pullUntilDone did not have a .catch or await try/catch on it's return value.
-    this.promise.catch(() => {});
+    this.promise.catch(function() {
```

Comment:
Was this a change as part of linting? Just curious what the rule was - or was it just that you needed to have something in the body?

Reply:
TypeScript Eslint plugin doesn't like fat arrows, then there's another rule to prevent us from having empty blocks.

Reply:
Can you elaborate more on the dislike of fat arrow? I'm fine commenting empty blocks, but fat arrow should be allowed.

Reply:
I'll try that.

Reply:
Worked! Thanks!